service-python/pdfgenerator/src/lib/templates/hypertension/about.html

```diff
@@ -1,4 +1,4 @@
-<h3>About this Document</h3>
+<h1 style="font-size: 21px; margin-top: 24px;">About this Document</h1>
```

Comment:
@yoomlam missed pushing a commit. Now the spacing is in place. Not sure theres a way to verify outside of comparing both PDFs visually or checking the code so it has a `margin-top: 24px`